roundabout-core/src/main/java/at/fhv/itm3/s2/roundabout/entity/DriverBehaviour.java

```diff
@@ -3,4 +3,41 @@
 import at.fhv.itm3.s2.roundabout.api.entity.IDriverBehaviour;
 
 public class DriverBehaviour implements IDriverBehaviour {
+
+    private double speed;
+    private double minDistanceToNextCar;
+    private double maxDistanceToNextCar;
+    private double behaviourFactor;
```

Comment:
mergeFactor in entity diagram, whats the purpose of this value?

Reply:
also, to be within our coding guidelines, i would advise to use the _ as prefix of a private variable, like _speed;

Reply:
no, this is bad practice: https://google.github.io/styleguide/javaguide.html